access-mgmt-service-registry/README.adoc

```diff
@@ -0,0 +1,143 @@
+:access-mgmt-url: access-mgmt/README.adoc
+:access-mgmt-service-registry-url: access-mgmt-service-registry/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-service-registry-access"]
+= Managing account access in {registry-product-long}
+ifdef::context[:parent-context: {context}]
+:context: managing-access-service-registry
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As an owner of a {registry} instance, administrator, or organization administrator in {registry-product-long}, you can manage the level of access that other user accounts and service accounts have to your {registry} instance. You can grant or remove access to your instance for specific user accounts in your organization based on user roles. You can also allow other users or service accounts to manage the level of access to your instance for you.
+
+{registry} instance owners or administrators can manage access for only the {registry} instances that they create or for instances that the owner has allowed them to access and change. Organization administrators can manage access for all {registry} instances.
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
+
+[id="con-registry-access-management_{context}"]
+== Access management in {registry-product-long}
+
+[role="_abstract"]
+{registry-product-long} uses Role-Based Access Control (RBAC) to manage how other user accounts and service accounts access the {registry} instances that you create and the artifacts that they contain. You can manage access for only the {registry} instances that you create or for instances that the owner has allowed you to access and change.
+
+An account in {registry-product-long} is either a user account or a service account. A user account enables users in your organization to access your {registry} instances. A service account enables your client application or tool to connect securely and to access your {registry} instances.
+
+You can also allow other users or service accounts to manage access to your {registry} instances for you.
+
+=== Supported user roles in {registry-product-long}
+
+The {registry} web console provides an *Access* tab on the {registry} instance page. {registry} instance owners, administrators, and organization administrators can use this tab to manage the following user roles:
+
+* *Administrator* - Can view or write user roles and schema and API artifacts, configure global rules for compatibility and validity, and import/export {registry} data in this {registry} instance 
+* *Manager* - Can view or write schema and API artifacts in this {registry} instance
+* *Viewer* - Can view schema and API artifacts in this {registry} instance
+
+NOTE: The owner of a {registry} instance has the administrator role for that instance by default, and can assign roles in the same organization. Other users in the organization have read-only viewer access by default.
```

Comment:
The second sentence here is not true.  Other users in the organization have **no access** by default.

Reply:
@EricWittmann Good spot! Updated and promoted the NOTE to an IMPORTANT :)